### PR TITLE
chore(ci): update schema generate examples

### PR DESCRIPTION
**Pull request by ocobles**
we need https://github.com/equinix/pulumi-equinix/pull/159 merged before this one to make sure that no new changes are discovered during the release process (the `toolchain go1.22.6` in go.mod in the examples depends on the go version installed locally. In my case I have 1.22.5 while in the GH action we have 1.22.x so it can take a newer version)

### CODE REVIEW

**Review by ctreatma — DISMISSED** [review]
Merging this ahead of #159 because we can validate this change by re-running the provider upgrade workflow after merge.  We can still hold off on releasing until the release workflow is updated.

**Comment by ctreatma** [comment]
Dismissed my own approval because I noticed CI failures and they seem relevant.  The `build_sdk` jobs are failing due to a dirty git tree: https://github.com/equinix/pulumi-equinix/actions/runs/10476042500/job/29014335287?pr=160#step:19:35

**Comment by ocobles** [comment]
> Dismissed my own approval because I noticed CI failures and they seem relevant. The `build_sdk` jobs are failing due to a dirty git tree: https://github.com/equinix/pulumi-equinix/actions/runs/10476042500/job/29014335287?pr=160#step:19:35

of course we need also a `make build_sdks`

**Review by ctreatma — APPROVED** [review]

**Pull request merged**
